packages/block-library/src/group/edit.js

```diff
@@ -48,23 +79,27 @@ function GroupEdit( { attributes, setAttributes, clientId } ) {
 	return (
 		<>
 			<InspectorControls __experimentalGroup="advanced">
-				<SelectControl
-					label={ __( 'HTML element' ) }
-					options={ [
-						{ label: __( 'Default (<div>)' ), value: 'div' },
-						{ label: '<header>', value: 'header' },
-						{ label: '<main>', value: 'main' },
-						{ label: '<section>', value: 'section' },
-						{ label: '<article>', value: 'article' },
-						{ label: '<aside>', value: 'aside' },
-						{ label: '<footer>', value: 'footer' },
-					] }
-					value={ TagName }
-					onChange={ ( value ) =>
-						setAttributes( { tagName: value } )
-					}
-				/>
+				<div className="block-library-group-block-html-element-control">
```

Comment:
Since you used `help`, we don't need this wrapper now and we can safely remove the extra styles in this PR as well.

Reply:
Good spots. I was somewhat rushing this at my EOD. My bad! Fixed.